roles/kube-gateway/templates/gateway.yml.j2

```diff
@@ -21,7 +21,7 @@ spec:
     - mountPath: /etc/kubernetes
       name: etc-kubernetes
 {% if datadog_enabled == true %}
-  - image: datadog/docker-dd-agent:latest
```

Comment:
Whoops, I missed this in my PR to move onto the `kubernetes` tag. Thank you for finding this